tests/ipfwd/test_nhop_group.py

```diff
@@ -749,3 +749,89 @@ def built_and_send_tcp_ip_packet():
     for flow_count, nexthop_selected in recvd_pkt_result.items():
         pytest_assert(nexthop_map[flow_count] in nexthop_selected,
                       "Flow {} is not picking expected Neighbor".format(flow_count))
+
+
+def test_nhop_group_interface_flap(duthost, tbinfo, ptfadapter, gather_facts,
+                                   enum_rand_one_frontend_asic_index, fanouthosts):
+    """
+    Test for packet drop when route is added with ECMP and all ECMP member's
+    interfaces are down. Use kernel flag 'arp_evict_nocarrier' to disable ARP
```

Comment:
Can you explain why this kernel flag must be changed for this test?

Reply:
Updated the description about the usage of kernel flag.